### PR TITLE
update manager role. Add doc for definting and placing gateways. remo…

### LINKED ISSUE #74: How To:  define and place a multi-cluster gateway
Depends on #54 

This should introduce the idea of a multi-cluster gateway and then go on to explain how you can create and place a multi-cluster gateway. This doc should appear under the how-to section of https://docs.kuadrant.io/multicluster-gateway-controller/  

It should have "getting started" as a pre-req

### PR DESCRIPTION
**Pull request by maleck13**
…ve local file refs

closes #74 

Main item is the new doc to describe how to define and place a gateway 
two other small fixes included
- update perms for gatewayclass controller to be able to read params
- change ocm walkthrough not to reference folders that might not exist for some locally

### CODE REVIEW

File: docs/how-to/ocm-control-plane-walkthrough.md [hunk 1]
```diff
@@ -335,7 +335,7 @@ So now we have a working gateway with DNS and TLS configured. Let place this gat
 1. In `T2` window execute the following to see the gateway on the workload-1 cluster:
 
     ```bash
-    kind export kubeconfig --name=mgc-workload-1 --kubeconfig=$(pwd)/local/kube/workload1.yaml && export KUBECONFIG=$(pwd)/local/kube/workload1.yaml
+    kind export kubeconfig --name=mgc-workload-1 --kubeconfig=$(pwd)/workload1.yaml && export KUBECONFIG=$(pwd)/workload1.yaml
```
Comment (philbrookes) [inline docs/how-to/ocm-control-plane-walkthrough.md:338]: this doesn't look right, why have the kubeconfig files moved to the root directory?

Comment (maleck13) [reply docs/how-to/ocm-control-plane-walkthrough.md:338]: root directory? You mean of the repo? It is only for the walkthrough and the local/kube dir may not exist locally

Comment (maleck13) [reply docs/how-to/ocm-control-plane-walkthrough.md:338]: I can remove it if you want. Wasn't really a part of this task just thought I would pick up while it was in my head

**Review by philbrookes — APPROVED** [review]
/lgtm

**Comment by philbrookes** [comment]
looks like e2e failed on throttling: https://github.com/Kuadrant/multicluster-gateway-controller/actions/runs/6022142751/job/16336296734?pr=461#step:9:155

**Comment by mikenairn** [comment]
> looks like e2e failed on throttling: https://github.com/Kuadrant/multicluster-gateway-controller/actions/runs/6022142751/job/16336296734?pr=461#step:9:155 I've restarted the e2e job now.

It's going to fail again. You need rebase main onto this branch as the e2e workflow has changed and isn't compatible with the resources this branch is creating (Error from server (NotFound): managedzones.kuadrant.io "mgc-dev-mz-aws" not found)

**Review by philbrookes — APPROVED** [review]
/lgtm

**Pull request merged**